src/Types.ts

```diff
@@ -172,6 +176,8 @@ export interface Context {
   description?: string;
 
   project_id: string;
+
+  is_project_default: boolean;
```

Comment:
+1. We used to do this via tagging. Does this mean the tag-based approach for identifying the default context is no longer needed?

Reply:
I believe so yes. When looking at the code, you almost always find the default context by checking if its name is null. I actually did not see anywhere where the code check the tag.  If you know of a spot let me know.